pkg/model/planner.go

```diff
@@ -121,20 +157,41 @@ func NewWorkflowPlanner(path string, noWorkflowRecurse bool) (WorkflowPlanner, e
 	for _, wf := range workflows {
 		ext := filepath.Ext(wf.workflowFileInfo.Name())
 		if ext == ".yml" || ext == ".yaml" {
+
 			f, err := os.Open(filepath.Join(wf.dirPath, wf.workflowFileInfo.Name()))
 			if err != nil {
 				return nil, err
 			}
 
-			log.Debugf("Reading workflow '%s'", f.Name())
+			log.Debugf("Reading workflow '%s'", file.Name())
```

Comment:
Shouldn't it be `f.Name()`?

Reply:
Yeah, didn't notice the change in the syntax of debug file names. Ill resolve this